src/components/General/Layout.js

```diff
@@ -1,23 +1,43 @@
 import React from 'react';
 import styled from 'styled-components';
 import BackgroundIcons from '../../backgroundIcons.svg';
+import logo from '../../logo.svg';
 
 const StyledMain = styled.main`
   background-image: url(${BackgroundIcons});
-  min-height: 100%;
+  min-height: 100vh;
+`;
+
+const StyledHeader = styled.header`
+  width: 100%;
+  background: #5f9645;
```

Comment:
It's a good idea to use the theme. Just import above and do:

```suggestion
  background: ${theme.primary.main};
```